main.go

```diff
@@ -47,14 +55,32 @@ func init() {
 }
 
 func main() {
+	// metricsAddr is used by Prometheus to gather the NFD Operator resource usage data.
+	// The bind address tells Prometheus which port to scrape this data's metrics from.
 	var metricsAddr string
+
+        // enableLeaderElection should be set to 'disable' by default If we enable leader
+        // election, then only one node can run the controller manager and we will not
+        // have NFD Operator running on all nodes.
 	var enableLeaderElection bool
+
+        // probeAddr is responsible for the health probe bind address, where the health
+        // probe is responsible for determining liveness, readiness, and configuration 
+        // of the operator pods.
 	var probeAddr string
+
+	// The following 3 lines setup the CLI arguments that are used upon initilization of
+	// the operator
```

Comment:
Maybe just `Setup the CLI arguments`, or smth. If a fourth argument would be added (or one removed) then the editor would need to remember to update the comment, too 😄

Reply:
Thanks. I've changed this line to "Setup the CLI arguments"